Add shortcode content to OP Q3 new components (#25515)
DataDog/documentation: layouts/shortcodes/observability_pipelines/prerequisites/logstash.md

To use Observability Pipelines' Logstash source, you need the following information available:

> - Logstash address. The Observability Pipelines Worker listens on this bind address to receive logs from your applications. For example, `0.0.0.0:8088`. Later on, you configure your applications to send logs to this address.

Reviewer: ```suggestion
- Logstash address, such as `0.0.0.0:8088`. The Observability Pipelines Worker listens on this bind address to receive logs from your applications. Later on, you configure your applications to send logs to this address.
```
